packages/react-native/ReactCommon/react/renderer/textlayoutmanager/platform/ios/react/renderer/textlayoutmanager/RCTTextLayoutManager.mm

```diff
@@ -303,12 +305,12 @@ - (TextMeasurement)_measureTextStorage:(NSTextStorage *)textStorage
 
                 CGSize attachmentSize = attachment.bounds.size;
                 CGRect glyphRect = [layoutManager boundingRectForGlyphRange:range inTextContainer:textContainer];
-
+                CGFloat baseline = [layoutManager locationForGlyphAtIndex:range.location].y;
                 UIFont *font = [textStorage attribute:NSFontAttributeName atIndex:range.location effectiveRange:nil];
```

Comment:
This variable is unused now and is causing various iOS builds to fail (we do have `-Wall` everywhere)